.gitignore

```diff
@@ -1,3 +1,2 @@
-venv/
-.env
-
+env/
+node_modules
```

Comment:
I'd still like to keep the virtual env as venv here as well as keep the .env ignored here too.

Also node modules might be redundant, the client I think has a node focused ignore file already